app/views/docs/getting-started-for-android.phtml

```diff
@@ -0,0 +1,126 @@
+<?php
+$platforms = $this->getParam('platforms');
+$versions = [];
+
+foreach($platforms['client']['languages'] as $lang) {
+    $key = $lang['key'] ?? '';
+    $version = $lang['version'] ?? '';
+
+    if(!empty($key)) {
+        $versions[$key] = $version;
+    }
+}
+
+$version = (isset($versions['android'])) ? $versions['android'] : '';
+?>
+<p>Appwrite is a development platform providing you easy yet powerful API and management console to get your next project up and running quickly.</p>
+
+<p>This tutorial will help you start using Appwrite products and build your next project. Before starting, make sure you have followed the Appwrite <a href="/docs/installation">installation guide</a>, and you have an Appwrite server instance up and running on your host machine or server.</p>
+
+<h2>Create Your First Appwrite Project</h2>
+
+<p>Go to your new Appwrite console, and once inside, click the <i class="icon-plus"></i> icon in the top navigation header or on the <b>'Create Project'</b> button on your console homepage. Choose a name for your project and click create to get started.</p>
+
+<h2>Add your Android Platform</h2>
+
+<p>To init your SDK and start interacting with Appwrite services, you need to add a new Android platform to your project. To add a new platform, go to your Appwrite console, choose the project you created in the step before, and click the 'Add Platform' button.</p>
+
+<p>From the options, choose to add a new <b>Android</b> platform and add add your app <u>name</u> and <u>package name</u>, Your package name is generally the applicationId in your app-level build.gradle file. By registering your new app platform, you are allowing your app to communicate with the Appwrite API.</p>
+
+<h2>Get Appwrite Android SDK</h2>
+
+<p>First, add this to your root level build.gradle file: </p>
+<div class="ide" data-lang="groovy" data-lang-label="Groovy">
+  <pre class="line-numbers">
+  <code class="prism language-groovy" data-prism><?php echo $this->escape('repositories {
+    maven { 
```

Comment:
This appears to be the configuration to publish to maven. Not to fetch the dependency.

You will have to add `mavenCentral()` to the `repositories` section under `buildscript`

Then include the dependency 

`implementation 'io.appwrite.sdk-for-android:0.0.1'`

Reply:
Done!